templates/backup-pruner.sh

```diff
@@ -0,0 +1,14 @@
+#jinja2: trim_blocks:False
+#!/bin/bash
+# This script runs all of the backup pruners in parallel, returning the output
+# of only the failing pruners.
+# Check {{ JOB_LOG }} for a summary of the most recent pruner run.
+set -e
+
+jobs='{% for item in TARSNAPPER_JOBS %}
+{{ item.name }}{% endfor %}'
+
+# Buffer all of the pruner output; if the pruner fails, forward along the
+# output to include in a cron email.
+wrapper='output=$(/usr/local/sbin/tarsnap-{}.sh expire 2>&1) || echo "$output"'
+echo "$jobs" | tail -n +2 | parallel --no-notice -j0 --joblog {{ JOB_LOG }} $wrapper
```

Comment:
Alright, I've added more specific commentary that should better remind us about these things when we come back to it. :) I'll leave the refactoring for us to do at a later time as you mention.